docs/tutorials/python/tutorial_scripts/annotation.py

```diff
@@ -0,0 +1,75 @@
+"""
+Here is where you'll find the code for the Annotation tutorial.
+"""
+# Step 1: Add several annotations to stored files
+import os
+import synapseclient
+from synapseclient import File
+
+syn = synapseclient.login()
+
+# Retrieve the project ID
+my_project_id = syn.findEntityId(
+    name="My uniquely named project about Alzheimer's Disease"
+)
+
+# Retrieve the folders I want to annotate files in
+batch_1_folder_id = syn.findEntityId(
+    name="single_cell_RNAseq_batch_1", parent=my_project_id
+)
+
+print(f"Batch 1 Folder ID: {batch_1_folder_id}")
+
+
+# Define the annotations I want to set
+annotation_values = {
```

Comment:
The annotation values can be set during the initial upload.  not sure if we wanted to complicate the upload script

Reply:
So in this script I am actually showing both ways. Uploading a file with annotations being set starts on L52.